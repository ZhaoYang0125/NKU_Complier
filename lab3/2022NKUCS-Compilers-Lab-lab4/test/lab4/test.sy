int a;
a=0;
int b;
int c;