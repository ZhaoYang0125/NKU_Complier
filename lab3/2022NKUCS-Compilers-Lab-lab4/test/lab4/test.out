[DEBUG LAB4]: 	INT   	int	1	0	
[DEBUG LAB4]: 	ID    	a	1	4	0x5608e84b0400
[DEBUG LAB4]: 	SEMICOLON	;	1	5	
[DEBUG LAB4]: 	ID    	a	2	0	0x5608e84b0400
[DEBUG LAB4]: 	ASSIGN	=	2	1	
[DEBUG LAB4]: 	DECIMIAL	0	2	2	
[DEBUG LAB4]: 	SEMICOLON	;	2	3	
[DEBUG LAB4]: 	INT   	int	3	0	
[DEBUG LAB4]: 	ID    	b	3	4	0x5608e84b08e0
[DEBUG LAB4]: 	SEMICOLON	;	3	5	
[DEBUG LAB4]: 	INT   	int	4	0	
[DEBUG LAB4]: 	ID    	c	4	4	0x5608e84b0970
[DEBUG LAB4]: 	SEMICOLON	;	4	5	
